lib/v2/lovelive-anime/radar.js

```diff
@@ -3,11 +3,20 @@ module.exports = {
         _name: 'Love Live 官网',
         www: [
             {
-                title: '最新NEWS',
-                docs: 'https://docs.rsshub.app/anime.html#lovelive-anime',
-                source: ['/'],
+                title: '最新 NEWS',
+                docs: 'https://docs.rsshub.app/anime.html#lovelive-anime-love-live-guan-wang-zui-xin-news',
+                source: ['/', '/news'],
                 target: '/lovelive-anime/news',
             },
+            {
+                title: '分类 Topics',
+                docs: 'https://docs.rsshub.app/anime.html#lovelive-anime-love-live-guan-wang-fen-lei-topics',
+                source: ['/:abbr/topics/', '/:abbr/topics/?cat=:category', '/:abbr/topics.php', '/:abbr/topics.php?cat=:category'],
```

Comment:
`?cat=:category` [won't work](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze-source).
You need to use `URL` to match/extract searchParams